guides/v2.2/cloud/configure/setup-cron-jobs.md

```diff
@@ -104,29 +104,41 @@ The [auto-crons feature](#verify-cron-configuration) must be enabled on your {{s
 
 #### To add custom crons
 
-1. In your local development environment, edit the `.magento.app.yaml` file in the Magento `/app` directory.
+1. In your local development environment, edit the `.magento.app.yaml` file in the Magento `/app` directory. 
+
+1. The structure of `crons` section in the `.magento.app.yaml` is:
+   
+   ```yaml
+   crons:
+       <cron_name_1>:
+           spec: "<schedule_time>"
+           cmd: "<schedule_command>"
+       <cron_name_2>:
+           spec: "<schedule_time>"
+           cmd: "<schedule_command>"        
+   ```
 
 1. Add your custom cron code to the `crons` section in the file.
```

Comment:
I recommend adding the crons section format sample from above, inside this step.

```suggestion
1. Add your custom cron code to the `crons` section in the configuration file using the following format:
```

Reply:
> I recommend adding the crons section format sample from above, inside this step.

**done**

Thank you, @hguthrie !